google-cloud-graalvm-samples/google-cloud-graalvm-pubsub-sample/pom.xml

```diff
@@ -31,41 +28,23 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-pubsub</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>io.grpc</groupId>
-          <artifactId>grpc-netty-shaded</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
 
     <dependency>
       <groupId>io.grpc</groupId>
-      <artifactId>grpc-netty</artifactId>
+      <artifactId>grpc-netty-shaded</artifactId>
+      <!-- Remove this dep after libraries-bom is updated to offer this version -->
+      <version>1.32.1</version>
```

Comment:
Should this go into dependencyManagement instead?

Reply:
Done.